src/tokenizer/scope.rb

```diff
@@ -40,7 +40,7 @@ def add_function(name, signature = [], options = {})
       key = function_key name, signature
       @functions[key] = { name: options[:alias_of] || name, signature: signature }
 
-      aliases = Conjugator.conjugate(name) + (options[:aliases] || [])
+      aliases = [name, *options[:aliases]].map { |n| Conjugator.conjugate n } .reduce(&:+)
```

Comment:
Previously, we'd register the function name, its conjugated names, and then the aliases... But I realised the aliases weren't conjugated.

This line is kinda cool: it shows two ways of flattening stuff. First it combines `[name, *options[:aliases]]` which uses the splat operator (`*`) to flatten the aliases (it also removes null!). Then `map` with conjugations, and then flattens it again with reduce.